tests/integration/components/feed-item-test.js

```diff
@@ -9,7 +10,34 @@ test('it renders', function(assert) {
   // Set any properties with this.set('myProperty', 'value');
   // Handle any actions with this.on('myAction', function(val) { ... });
 
-  this.render(hbs`{{feed-item}}`);
+  let article = {
+    author: {
+      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
+      username: 'Alon Bukai'
+    },
+    createdAt: moment('1995-12-25'),
+    favoritesCount: 9000,
+    body: 'This is feed item content',
+    slug: 'feed-item-title',
+    title: 'Feed Item Title'
+  };
 
-  assert.equal(this.$().text().trim(), '');
+  this.set('article', article);
+  this.render(hbs`{{feed-item article=article }}`);
+
+  assert.equal(
+    this.$('img').attr('src'),
+    'https://static.productionready.io/images/smiley-cyrus.jpg',
+    'Image is default image'
+  );
+
+  assert.equal(this.$('a.author').text().trim(), 'Alon Bukai', 'Author name is correct');
```

Comment:
Now that we're actually writing tests (yay!), I think we should pull in something like [`ember-hook`](https://www.npmjs.com/package/ember-hook) so that we can avoid coupling test selectors to the shape of the DOM.  What do you think?